docker-compose/stacks/pocket-blockchain/docker-compose.yaml

```diff
@@ -0,0 +1,143 @@
+version: "3.8"
+
+# REF: https://www.cloudbooklet.com/how-to-install-nginx-and-lets-encrypt-with-docker-ubuntu-20-04/
+
+services:
+  web: 
+    image: nginx:alpine
+    restart: always
+    environment:
+      - DOMAIN=${DOMAIN}
+    volumes:
+      - ${PWD}/proxy/conf.d:/etc/nginx/templates
+      - ${PWD}/proxy/nginx.conf:/etc/nginx/nginx.conf
+      - ${PWD}/proxy/certbot/conf:/etc/nginx/ssl
+      - ${PWD}/proxy/certbot/data:/var/www/certbot
+    ports:
+      - 80:80
+      - 443:443
+    depends_on:
+      - prometheus
+      - grafana
+      - node1   
+
+  certbot:
+    image: certbot/certbot:latest
+    container_name: certbot
+    command: certonly --webroot --webroot-path=/var/www/certbot --email ${EMAIL} --agree-tos --no-eff-email -d node1.${DOMAIN} -d monitoring.${DOMAIN}
+    environment:
+      - DOMAIN=${DOMAIN}
+      - EMAIL=${EMAIL}
+
+    volumes:
+      - ${PWD}/proxy/certbot/conf:/etc/letsencrypt
+      - ${PWD}/proxy/certbot/logs:/var/log/letsencrypt
+      - ${PWD}/proxy/certbot/data:/var/www/certbot
+
+    depends_on:
+      - prometheus
+      - grafana
+      - node1
+      - web
+
+  node1:
+    image: poktnetwork/pocket:Beta-0.5.2.1
+    container_name: node1 
+    command: ["pocket", "start", "--keybase=false", "--mainnet", "--datadir=/home/app/.pocket/"]
+    ports: 
+      - "26656:26656"
+    expose:
+      - "8081"
+      - "26656"
+      - "26660"
+    volumes:
+      - ${PWD}/node1/genesis.json:/home/app/.pocket/config/genesis.json                    # Genesis 
+      - ${PWD}/node1/config.json:/home/app/.pocket/config/config.json                      # Pocket Configuration file 
+      - ${PWD}/node1/data/:/home/app/.pocket/data/                                         # Blockchain data
+    environment:
+      - POCKET_CORE_PASSPHRASE=''
+      - POCKET_CORE_KEY=''
+    logging:
+      driver: loki
+        options:
```

Comment:
Error in the indentation, should be at the same indentation as the `driver`.

Reply:
Thanks! @luyzdeleon fixed in https://github.com/pokt-network/pocket-core-deployments/pull/170/commits/aa859c26bc851e232ce8085770de044b120d3ac8